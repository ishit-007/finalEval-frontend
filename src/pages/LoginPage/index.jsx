import { InputCredentials, LoginScreenImage } from '../../components';
import React from 'react';

const LoginPage = () => {
  return (
    <div className="loginPage">
      <div className="loginPage__left">
        <LoginScreenImage />
      </div>
      <div className="loginPage__right">
        <InputCredentials />
      </div>
    </div>
  );
};

export default LoginPage;
