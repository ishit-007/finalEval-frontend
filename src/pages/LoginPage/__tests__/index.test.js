import LoginPage from '..';
import renderer from 'react-test-renderer';
import React from 'react';

jest.mock('react-router-dom');

describe('LoginPage', () => {
  it('should render the LoginPage component', async () => {
    const tree = renderer.create(<LoginPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});