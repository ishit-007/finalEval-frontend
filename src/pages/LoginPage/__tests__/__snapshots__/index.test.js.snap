// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should render the LoginPage component 1`] = `
<div
  className="loginPage"
>
  <div
    className="loginPage__left"
  >
    <div
      className="loginImage"
    >
      <img
        alt="Login Screen"
        src="undraw-upload-re-pasx@3x.png"
      />
      <div
        className="content"
      >
        <h1>
          Design API's fast,
        </h1>
        <h1>
          Manage content easily.
        </h1>
      </div>
    </div>
  </div>
  <div
    className="loginPage__right"
  >
    <div
      className="inputForm"
    >
      <div
        className="heading"
      >
        <h1>
          Login to your CMS+ Account
        </h1>
      </div>
      <div
        className="emailInput"
      >
        <label
          htmlFor=""
        >
          Email
        </label>
        <input
          id="email"
          name="email"
          onChange={[Function]}
          type="email"
        />
      </div>
      <div
        className="passwordInput"
      >
        <label
          htmlFor=""
        >
          Password
        </label>
        <input
          id="password"
          name="password"
          onChange={[Function]}
          type="password"
        />
      </div>
      <div
        className="submitBtn"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          Login
        </button>
        <a
          href=""
        >
          Forgot Password?
        </a>
        <a
          className="other-Option"
          href=""
          onClick={[Function]}
        >
          Register Instead
        </a>
      </div>
    </div>
  </div>
</div>
`;
